repairnator/repairnator-pipeline/src/main/java/fr/inria/spirals/repairnator/pipeline/Launcher.java

```diff
@@ -108,9 +111,9 @@ private void initConfig() {
         this.config.setLauncherMode(LauncherMode.valueOf(this.arguments.getString("launcherMode").toUpperCase()));
         this.config.setClean(true);
         this.config.setZ3solverPath(this.arguments.getFile("z3").getPath());
-        if (this.arguments.getFile("outputPath") != null) {
+        if (this.arguments.getFile("output") != null) {
```

Comment:
if you defined them in the LauncherUtils, you should get access to them through a static method from LauncherUtils.

Reply:
Done.